pkgs/development/python-modules/python-arango/default.nix

```diff
@@ -0,0 +1,139 @@
+{ lib
+, arangodb
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, pyjwt
+, pytest
+, mock
+, requests
+, requests-toolbelt
+}:
+
+let
+  testDBOpts = {
+    host = "127.0.0.1";
+    port = "8529";
+    password = "test";
+    secret = "secret";
+  };
+in
+
+buildPythonPackage rec {
+  pname = "python-arango";
+  version = "7.5.1";
+  disabled = pythonOlder "3.7";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "ArangoDB-Community";
+    repo = "python-arango";
+    rev = version;
+    sha256 = "0qyf2vc95ah6zscim3ra9ifn3lzjh845681dnlcj0q84d3z14510";
+  };
+
+  # arangodb is compiled only for particular target architectures
+  # (i.e. "haswell"). Thus, these tests may not pass reproducibly,
+  # failing with: `166: Illegal instruction` if not run on arangodb's
+  # specified architecture.
```

Comment:
Then we should limit the package to those platforms or only the tests instead of disabling them completely.

Reply:
I would like to do that, but I don't know how. How would you go about it?

Reply:
@SuperSandro2000 Just for some background, the architectures that arango is sensitive to are not (i.e.) x86 vs aarch, it is haswell vs sandy-bridge.

Reply:
Thats a problem because we have no system to detect this that does not require a mass rebuild. Does it work if we run the tests on the same architecture we compile? Can we force an older architecture like haswell?